src/components/Input/Input.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styles from "./Input.module.css";
+
+interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  error?: boolean;
+}
+
+const Input = React.forwardRef<HTMLInputElement, InputProps>(
+  ({ error, className = "", ...rest }, ref) => {
+    return (
+      <input
+        ref={ref}
+        className={`${styles.input} ${error ? styles.errorInput : ""} ${className}`}
+        {...rest}
+      />
+    );
+  },
+);
```

Comment:
```suggestion
const Input = ({ error, className = "", ...rest }: InputProps) => {
  return (
    <input
      className={`${styles.input} ${error ? styles.errorInput : ""} ${className}`}
      {...rest}
    />
  );
};
```
React 19버전 부터는 `ref` 속성이 일반 `Prop`로 병합되어서 `forwardRef`를 사용하지 않고도 `ref` 속성을 넘겨줄 수 있을 것 같아요!